lib/Parser/source.cpp

```diff
@@ -111,36 +112,36 @@ static std::size_t RemoveCarriageReturns(char *buffer, std::size_t bytes) {
   return wrote;
 }
 
-bool SourceFile::Open(std::string path, std::stringstream *error) {
+bool SourceFile::Open(std::string path, llvm::raw_ostream &error) {
   Close();
   path_ = path;
   std::string errorPath{"'"s + path + "'"};
   errno = 0;
   fileDescriptor_ = open(path.c_str(), O_RDONLY);
   if (fileDescriptor_ < 0) {
-    *error << "Could not open " << errorPath << ": " << std::strerror(errno);
+    error << "Could not open " << errorPath << ": " << std::strerror(errno);
```

Comment:
std::strerror -> llvm::sys::StrError? 
A few other occurences as well.

Reply:
So, the compiler did not complain about that, and I believe it belongs to #include <cerrno>. Moreover, there is nothing in the LLVM Code style talking about this function.
But maybe you are right because I do see llvm::sys::StrError being used in llvm-project.
@DavidTruby ??

Reply:
Ok, so it looks there is nothing asking to change, but the llvm one is thread-safe. So, for this reason(being thread-safe)[1], I am replacing std::strerror by  llvm::sys::StrError.
[1]https://llvm.org/doxygen/Errno_8cpp_source.html

Reply:
llvm::sys::StrError has the same visible behaviour, but is thread safe where possible. I think we should use it over std::strerror for that reason.

Reply:
Done!